schemas/v0.1/entries.schema.json

```diff
@@ -96,7 +96,7 @@
                                             "number",
                                             "string"
                                         ],
-                                        "description": "When epoch is needed, specify as number. String should be only used for \"now\", meaning the acquisition date."
+                                        "description": "When epoch is needed, specify as number. String should be only used for \"now\", meaning the acquisition date, or \"mid-year\", meaning the middle of the current year (20xx.5)"
```

Comment:
I think we need to change this to be more strict using `oneOf` to choose between `number` and `enum` of type `string` with `"now"` and `"mid-year"`.
Something like:
```
"epoch": {
 "oneOf": [
  {
   "type": "number"
  },
  {
   "type": "string"
   "enum": ["now", "mid-year"]
  }
 ]
}   
```
(to be completed with descriptions)

Reply:
Agreed, still getting the feel for the JSON schema spec, haven't used it before. Added!